pkg/authorization/api/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package api is the internal version of the API.
+// +genconversion=true
```

Comment:
And I guess we will have to change **all** defaulters to use the new `SetDefaults_FOO` scheme /o\